tools/mac_setup.sh

```diff
@@ -52,6 +53,7 @@ brew "zeromq"
 brew "protobuf"
 brew "protobuf-c"
 brew "swig"
+brew "rustup-init"
```

Comment:
rust is pretty big, don't want to include it just for acados since that's checked in

Reply:
I’ll move the install to the relevant part in build.sh of thirdparty/acados then